test/python/test_pack.py

```diff
@@ -264,6 +243,31 @@ def test_packing_without_git(tmpdir):
         project_path,
     ]
     process = subprocess.run(cmd, cwd=tmpdir)
+    assert process.returncode == 0
+    assert 'empty-project-0.1.0-0.rpm' in os.listdir(tmpdir)
```

Comment:
```python
project_name = project_without_dependencies['name']
assert '{}-0.1.0-0.rpm'.format(project_name) in os.listdir(tmpdir)
```

Just to prevent unexpected test failure on changing `project_without_dependencies` name.

Reply:
Yes, thanks, fixes